src/package/packageVersionCreate.ts

```diff
@@ -344,14 +353,17 @@ export class PackageVersionCreate {
         throw messages.createError('signupDuplicateSettingsSpecified');
       }
 
-      ['country', 'edition', 'features', 'orgPreferences', 'snapshot', 'release', 'sourceOrg'].forEach((prop) => {
-        // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
-        const propValue = definitionFileJson[prop];
-        if (propValue) {
+      // these scratch org definition file values will be applied to the build org
+      ['country', 'language', 'edition', 'features', 'orgPreferences', 'snapshot', 'release', 'sourceOrg'].forEach(
```

Comment:
@sethmuss - this adds back the `language` from the scratch org def file like it was before when we had that bug (before 242 R2).  I thought you said that the language set for the scratch org should not be applied here?

Reply:
@shetzel Yes. I said that because I didn't fully understand the use-case for it to be present in the scratch/build org for a package version create. A customer ran into an issue exposing that having the language in the scratch/build org is something we still need to support. This looks right to me. We have language on the Package2VersionCreateRequest on line 258, and language into the descriptor for the scratch/build org support. Sorry for the confusion Steve, that's my bad.

Reply:
Ok thanks for the verification @sethmuss!